zrepl.json

```diff
@@ -1,6 +1,6 @@
 {
     "name": "zrepl",
-    "plugin_schema": 2,
```

Comment:
@fulder let's remove this entirely, it's not being used anymore

Reply:
That's true, can't see any references att all in `iocage` repo. Have removed it now.